Import css/css-values WPT from revision f15017ebef88a25582f2c711d40d7649e692dfa4

nt1m opened the pull request:
#### cbda0d352a77c4af125f3d39934bd7ab15cd564f
<pre>
Import css/css-values WPT from revision f15017ebef88a25582f2c711d40d7649e692dfa4
<a href="https://bugs.webkit.org/show_bug.cgi?id=242470">https://bugs.webkit.org/show_bug.cgi?id=242470</a>

Reviewed by Simon Fraser.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/absolute-length-units-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/absolute-length-units-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/calc-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/calc-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-angle-values-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-angle-values.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-ch-ex-lang-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-in-counter-001-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-in-media-queries-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-range-clamping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-bottom-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-bottom-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-left-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-left-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-right-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-right-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-bottom-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-bottom-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-right-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-right-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-positive-fraction-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rem-lang-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rgb-percent-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rounds-to-integer-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rounds-to-integer.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-z-index-fractions-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/cap-unit-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/cap-unit-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-empty-pseudo-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-pseudo-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/dynamic-viewport-units-rule-cache-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/dynamic-viewport-units-rule-cache.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-calc-expression-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-unit-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-unit-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-008-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-010-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-013.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-014.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/initial-background-color-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/initial-background-color.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-percentage-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-percentage-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/q-unit-case-insensitivity-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/q-unit-case-insensitivity-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rgba-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-mod-rem-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-mod-rem-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/blue-32x32.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/unset-value-storage-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/unset-value-storage.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-after-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-compute-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-compute.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-keyframes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-keyframes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-media-queries-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-media-queries.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-modify-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-modify.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/w3c-import.log:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/252256@main">https://commits.webkit.org/252256@main</a>
</pre>

smfr commented on LayoutTests/TestExpectations:
```diff
@@ -3424,6 +3424,11 @@ webkit.org/b/203331 imported/w3c/web-platform-tests/css/css-values/attr-px-valid
 webkit.org/b/203333 imported/w3c/web-platform-tests/css/css-values/ch-unit-003.html [ ImageOnlyFailure ]
 webkit.org/b/203333 imported/w3c/web-platform-tests/css/css-values/ch-unit-010.html [ ImageOnlyFailure ]
 webkit.org/b/203338 imported/w3c/web-platform-tests/css/css-values/vh_not_refreshing_on_chrome.html [ ImageOnlyFailure ]
+webkit.org/b/242465 imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1.html [ Skip ]
```
Why Skip and not Fail?

nt1m replied:
It's a crash, so I guess I could put Crash, but I think that's usually avoided given it takes a bit more time.

nt1m committed:
Import css/css-values WPT from revision f15017ebef88a25582f2c711d40d7649e692dfa4
https://bugs.webkit.org/show_bug.cgi?id=242470

Reviewed by Simon Fraser.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/absolute-length-units-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/absolute-length-units-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/calc-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/calc-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-angle-values-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-angle-values.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-image-gradient-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-linear-gradient-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-position-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-background-size-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-border-radius-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-ch-ex-lang-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-height-table-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-in-counter-001-ref.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-in-media-queries-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-range-clamping.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-margin-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-max-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-min-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-bottom-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-bottom-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-left-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-left-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-right-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-right-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-absolute-top-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-bottom-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-bottom-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-left-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-right-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-right-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-offsets-relative-top-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-padding-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-positive-fraction-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rem-lang-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rgb-percent-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rounds-to-integer-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-rounds-to-integer.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-text-indent-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-transform-origin-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-vertical-align-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-block-intrinsic-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-auto-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-width-table-fixed-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-z-index-fractions-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/cap-unit-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/cap-unit-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-empty-pseudo-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-pseudo-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-recalc-on-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-004.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ch-unit-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/dynamic-viewport-units-rule-cache-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/dynamic-viewport-units-rule-cache.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-calc-expression-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-unit-001-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ex-unit-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-008-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-008.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-009-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-009.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-010-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-010.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-011-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-012-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-013.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/ic-unit-014.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/initial-background-color-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/initial-background-color.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-percentage-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-percentage-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/negative-calc-to-non-negative-integer.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/q-unit-case-insensitivity-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/q-unit-case-insensitivity-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rem-root-font-size-restyle-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rgba-011.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-mod-rem-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-mod-rem-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/blue-32x32.png: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/unset-value-storage-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/unset-value-storage.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-001-print.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-after-font-load.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-compute-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-compute.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-keyframes-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-keyframes.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-media-queries-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-media-queries.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-modify-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-modify.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode-font-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/viewport-units-writing-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/w3c-import.log:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-values/lh-rlh-on-root-001-expected.txt: Removed.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/252256@main
